yb-voyager/src/utils/gcs/gcsutils.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright (c) YugabyteDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package gcs
```

Comment:
Someday, we should get rid of these `{gcs,s3,az}utils.go` by folding their code in the corresponding *Datastore.go files.

For now, let's go with what is implemented here.